shared/css/color.scss

```diff
@@ -73,6 +73,7 @@ $realyellow: #ff0;
 $mustardyellow: #efcd1c;
 $twitter_blue: #00aced;
 $facebook_blue: #3b5998;
+$linkedin_blue: #0077B5;
```

Comment:
The figma has this color as `#007BB5` but [LinkedIn's official guidelines](https://brand.linkedin.com/content/brand/global/en_us/index/visual-identity/color-palettes) say to use `#0077B5`, so I went with that.